pycbc/io/hdf.py

```diff
@@ -729,7 +734,13 @@ def __init__(self, coinc_file, bank_file, sngl_files=None, n_loudest=None,
     @property
     def sort_arr(self):
         if self._sort_arr is None:
-            ifar = self.coinc_file.get_column('ifar')
+            if self._inclusive:
+                try:
+                    ifar = self.coinc_file.get_column('ifar')
+                except KeyError:
+                    logging.warning("WARNING: Can't find inclusive IFAR!")
+                    ifar = self.coinc_file.get_column('ifar_exc')
+                    self._inclusive = False
```

Comment:
The logic here doesn't look right. If `self._inclusive` is False, then `ifar` would not be set and the code will fail on the line below.

Reply:
Ah, I clearly wasn't anticipating anyone would be advanced enough to actually set _inclusive=False from the start ...